plugins/core/gs-appstream.h

```diff
@@ -71,5 +71,7 @@ void		 gs_appstream_component_add_icon	(XbBuilderNode	*component,
 							 const gchar	*str);
 void		 gs_appstream_component_add_provide	(XbBuilderNode	*component,
 							 const gchar	*str);
+void		 gs_appstream_component_add_kudo	(XbBuilderNode	*component,
```

Comment:
Shouldn’t this new function be called from somewhere?

Reply:
Sorry, I made a `fixup/squash/rebase` fiasco and had hunks not pushed on the PR. 
Pushed now.